SCClassLibrary/Common/GUI/PlusGUI/Math/PlotView.sc

```diff
@@ -918,10 +980,10 @@ Plotter {
 
 	plotColor_ { |colors|
 		plotColor = colors.as(Array);
-		plots.do { |plt, i|
+		plots.do { |plot, i|
```

Comment:
This was in response to @telephon's feedback that dropping just a couple letters may not be worth the brevity. And looking through this file, other methods like `makePlots`, `updatePlots`, `updatePlotSpecs`, etc. use the `plot` arg.

Reply:
Thanks! I understand the comment about dropping letters, but I'd presume this to apply to new argument names. Changing old ones is always tricky - will it break others' code? It might... but for single args maybe that's unlikely enough?

Reply:
in this case the arg is internal to the loop over `plots`, so users' code won't touch it.

Reply:
Ugh... my bad, sorry!